src/main/java/com/vlf/intentsender/adb/AdbHelper.java

```diff
@@ -1,12 +1,12 @@
-package intentsender.adb;
+package com.vlf.intentsender.adb;
 
 import com.android.ddmlib.*;
 import com.android.tools.idea.sdk.AndroidSdks;
 import com.intellij.ide.util.PropertiesComponent;
 import com.intellij.openapi.project.Project;
-import intentsender.Models.Command;
-import intentsender.Models.ExtraField;
-import intentsender.Models.IntentFlags;
+import com.vlf.intentsender.Models.Command;
```

Comment:
Is that expected that `Models` folder/package name is capitalized?

Reply:
That's the original packaging. I do not like that as well but may be a part of separate PR. Thanks for noticing

Reply:
Updated. It is lowercase now. Thanks 🍻